src/vs/workbench/contrib/debug/test/browser/debugANSIHandling.test.ts

```diff
@@ -110,29 +110,39 @@ suite('Debug - ANSI Handling', () => {
 	 * @param element The HTML span element to look at.
 	 * @param colorType If `foreground`, will check the element's css `color`;
 	 * if `background`, will check the element's css `backgroundColor`.
+	 * if `underline`, will check the elements css `textDecorationColor`.
```

Comment:
Thanks for writing tests.

Reply:
Well the tests of course took several times longer to code than the features.. but that's always the case isn't it ;) ?

Reply:
Yeah :)